python/tach/__init__.py

```diff
@@ -0,0 +1 @@
+from __future__ import annotations
```

Comment:
Is this newly required/does this apply to all of our python code?

Reply:
good catch, leftover from testing invalid imports